src/compiler/commandLineParser.ts

```diff
@@ -669,6 +677,100 @@ namespace ts {
     }
 
     /**
+     * Generate tsconfig configuration when running command line "--init"
+     * @param options commandlineOptions to be generated into tsconfig.json
+     * @param fileNames array of filenames to be generated into tsconfig.json
+     */
+    /* @internal */
+    export function generateTSConfig(options: CompilerOptions, fileNames: string[]): { compilerOptions: Map<CompilerOptionsValue> } {
+        const compilerOptions = extend(options, defaultInitCompilerOptions);
+        const configurations: any = {
+            compilerOptions: serializeCompilerOptions(compilerOptions)
+        };
+        if (fileNames && fileNames.length) {
+            // only set the files property if we have at least one file
+            configurations.files = fileNames;
+        }
+        else {
+            configurations.exclude = ["node_modules"];

```

Comment:
no. a tsconfig.json with no excludes property is treated as a tsconfig.json with an exclude property with `node_modules` and `outDir`. That was not the behavior when we added support for `--init`, so this was needed, but not now. so the exclude property in the generated tsconfig.json is currently superfluous.